src/security.md

```diff
@@ -0,0 +1,33 @@
+Securing a Guacamole install
+============================
+
+:::{toctree}
+:hidden:
+
+SSL termination <reverse-proxy>
+Blocking brute-force attacks <auth-ban>
+:::
+
+Secure deployment of Guacamole requires that all communication between users
+and Guacamole are encrypted using SSL/TLS. We generally recommend using a reverse proxy like Apache HTTPD or Nginx and configuring that proxy to provide SSL termination:
```

Comment:
Might want to split this line, consistent with the style throughout the rest of the manual...